Doc/whatsnew/2.3.rst

```diff
@@ -8,7 +8,7 @@
 
 .. $Id: whatsnew23.tex 54631 2007-03-31 11:58:36Z georg.brandl $
 
-This article explains the new features in Python 2.3.  Python 2.3 was released
+This article explains the new features in Python 2.3. Python 2.3 was released
```

Comment:
Similarly, double spacing was commonly used in documents in the past (and still is by some authors) -- see the remainder of What's New in Python 2.3 where `.  ` (full stop with two literal space characters) is used frequently. Please revert this change.

A

Reply:
& also please revert where you've changed a double space to a single space in other documents (e.g. 3.0).

Reply:
Reverted mentioned double space. 3.0 release date was added later because it was missing. But it's adjusted to double spacing now.